cray/modules/rrs/swagger.yaml

```diff
@@ -0,0 +1,819 @@
+#
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# Rack Resiliency Service API Specification
+openapi: 3.1.1
+info:
+  title: Rack Resiliency Service
+  license:
+    name: Hewlett Packard Enterprise Development LP
+    url: http://www.hpe.com/
+  version: 1.0.0
+  description: >
+    The Rack Resiliency Service (RRS) provides a set of APIs to manage Rack Level resiliency. It queries the Kubernetes cluster to provide aggregated zone
+    information and detailed critical service status. It gathers node details across various zones
+    and presents both high-level summaries and in-depth information for zones and critical services.
+
+    ## Resources
+
+      ### GET /zones
+        Retrieve aggregated zone configuration showing Kubernetes Topology Zones and CEPH Zones including:
+          - Management Master Nodes
+          - Management Storage Nodes
+          - Management Worker Nodes
+          
+        Alternatively, if zones are not configured, one of the following informational messages is returned:
+          - "No K8s Topology/Ceph Zones configured"
+          - "No Ceph zones configured"
+          - "No K8s topology zones configured"
+
+      ### GET /zones/{zone_name}
+        Retrieve detailed information for a specific zone including:
+          - Zone Name
+          - Management Masters
+          - Management Workers
+          - Management Storages
```

Comment:
Up above in lines 41-43 it has `Management Master Nodes` etc., but here it is `Management Masters`.  Is that intentional to use two different names?

Also, we really should be switching away from calling them `Master` nodes, and instead calling them `Control Plane` nodes, that transition started back in k8s 1.20 and is was completed by k8s 1.27, we're now at k8s 1.32 in CSM 1.7.  One could use `Management Control Plane (Master) Nodes`, but maybe that's a bit wordy.  Anyway, use your judgement on whether or not you want to pick up the newer `Control Plane` term.